### PR TITLE
#165952876 Fix subscriptions

### PR DESCRIPTION
**Pull request by IssaIan**
### What does this PR do?
- Fix user subscriptions
### What are the tasks to be completed?
* Enable users to unsubscribe from in app notifications without affecting email notifications
* Fix message returned when no notifications are found
### Local Development Setup
Refer to the project Readme.md for this.
**Test with postman**
1. Signing up two users and verify
2. Follow one user: 
```
POST  http://127.0.0.1:8000/api/profiles/<*username*>/follow/
```
3. Create an article using the followed user:
```
POST http://127.0.0.1:8000/api/articles/
```
4. Create a comment with the followed user:
```
POST http://127.0.0.1:8000/api/articles/<*slug*>/comments/
```
5. Fetch the following-user notifications using:
```
GET http://127.0.0.1:8000/api/notifications/
```
**Screenshots**
Screenshot 2019-05-11 at 15 18 55

### What are the relevant pivotal tracker stories?
[#165305765](https://www.pivotaltracker.com/n/projects/2326460/stories/165952876)

### CODE REVIEW

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

**Review by Amoswachira — APPROVED** [review]

**Review by a-braham — APPROVED** [review]
LGTM

File: authors/apps/appnotifications/urls.py [hunk 1]
```diff
@@ -23,5 +23,10 @@
         'notifications/unsubscribe_email/<str:token>/',
         views.UnsubscribeEmailAPIView.as_view(),
         name="opt_out_link"
+    ),
+    path(
+        'notifications/delete/',
```
Comment (oluwagbenga-joloko) [inline authors/apps/appnotifications/urls.py]: why is this URL named as such `notifications/delete/`  and not just `notifications/`? I think Endpoints should be named based on the resources they interact with and not HTTP Verbs.

Comment (IssaIan) [reply authors/apps/appnotifications/urls.py]: Made the change @oluwagbenga-joloko

**Pull request merged**